### ISSUE TITLE
Improve resizing experience

### ISSUE THREAD

**Issue by humdingerb** [HaikuArchives/TheAwesomeResizer#5]
Resizing the image window with locked aspect ratio is a bit weird. Try pulling on the resizing-corner and make the image smaller and bigger a few times: The mouse pointer gets further and further away from the resize-corner.
Can the mouse pointer be locked onto the resize-corner?

**Comment by janus2** [comment]
I don't think so even Workspaces has the same problem and is around since ever...
